### DISCUSSION TITLE
Schedule django management command

### DISCUSSION THREAD

**Discussion by ghost** [logandk/serverless-wsgi#211] [Q&A]
How can I schedule a custom django management command?

I want to run 'command_x' every 5 minutes. Any ideas?

**Comment by ahasna** [comment]
If you are hosting your code using AWS Lambda, you can use AWS EventBridge (aka Cloudwatch Events) to invoke your function every 5 minutes and let your desired script run on startup. Another approach is to have your cronjobs live in a different lambda other than the one hosting your API Logic
